confidant/app.py

```diff
@@ -1,33 +1,62 @@
 import logging
 
 import boto3
+import guard
 from flask import Flask
 from flask_sslify import SSLify
 
 from confidant import settings
-
+from confidant.routes import (
+    blind_credentials,
+    credentials,
+    identity,
+    saml,
+    services,
+    static_files,
+)
 
 if not settings.get('DEBUG'):
     boto3.set_stream_logger(level=logging.CRITICAL)
     logging.getLogger('botocore').setLevel(logging.CRITICAL)
     logging.getLogger('pynamodb').setLevel(logging.WARNING)
 
-static_folder = settings.get('STATIC_FOLDER')
+CSP_POLICY = {
+    'default-src': ["'self'"],
+    'style-src': [
+        "'self'",
+        "'unsafe-inline'"  # for spin.js
+    ]
+}
+
+
+def create_app():
+    static_folder = settings.STATIC_FOLDER
+
+    app = Flask(__name__, static_folder=static_folder)
+    app.config.from_object(settings)
+    app.config.update(settings.encrypted_settings.get_all_secrets())
+    app.debug = settings.DEBUG
+
+    if settings.SSLIFY:
+        sslify = SSLify(app, skips=['healthcheck'])  # noqa
```

Comment:
does the var `sslify` need to be used elsewhere?  Looks like it's unreferenced at moment

Reply:
Ah, yeah, I could probably change this to just `SSLify(app, skips=['healthcheck'])` Let me update that.